components/dropzone/stories/template.js

```diff
@@ -17,32 +19,69 @@ export const Template = ({
 	title,
 	description,
 	label,
-	id = getRandomId("dropzone"),
-} = {}, context = {}) => {
-	return html`
-		<div
-			class=${classMap({
+	id,
+	size = "m",
+	image = "dropzone-illustration.png",
+	...globals
+}) => html`
+	<div
+		class=${classMap({
 			[rootClass]: true,
 			"is-dragged": isDragged,
 			"is-filled": isFilled,
 			...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+		})}
+		id=${ifDefined(id)}
+		role="region"
+		tabindex="0"
+		style=${[isDragged && isFilled ? `background-image:url(${image});` : null, styleMap(customStyles)].filter(Boolean).join(" ")}
```

Comment:
I'm seeing this `[object Object]` in the inspector, does something here need to be adjusted?
![image](https://github.com/user-attachments/assets/8afe15e7-ed3b-4921-8165-2e2cd174501f)